src/cart/crt_internal_types.h

```diff
@@ -170,6 +171,8 @@ struct crt_gdata {
 	long			 cg_num_cores;
 	/** Inflight rpc quota limit */
 	uint32_t		cg_rpc_quota;
+	/** number of HG_Init_opt2() when using CXI provider */
```

Comment:
guessing missing 'retries of' after 'number of'

Reply:
fixed. sorry for the force push